sw/device/silicon_creator/rom/data/rom_testplan.hjson

```diff
@@ -948,8 +948,13 @@
       name: rom_e2e_asm_init
       desc: '''Verify that ROM initializes peripherals properly.
 
-            Note: While the following can be checked at ROM_EXT stage, using a debugger would avoid
-            depending on `rom.c` for this test. Also, we may want to break this into multiple tests.
+            Note: While the following can be checked at ROM_EXT stage, using a debugger in
+            FPGA-based tests would avoid depending on `rom.c`. Doing so would give us more
+            information and help us narrow down possible issues we end up having a problem in ROM.
+            For DV, we don't have to use the JTAG interface: we can wait until Ibex accesses
+            `rom_main()` and backdoor load the values we want to check.
```

Comment:
nit:

```suggestion
            `rom_main()` and backdoor check specific values.
```
?

Reply:
we are checking values, not loading them right? or am I missing something?

Reply:
yes, I think I was trying to say something like "load from the simulation to testbench to check" or something :)
done